new handheld game Treasure Island (Tronica)

MilanBA opened the pull request:
New working systems
-------------------
Treasure Island (Tronica) [Milan Galcik, hap, Henrik Algestam]

happppp commented:
Good to see you here Milan.
When adding a new game, if you haven't already done so, a reminder to submit the dump here: https://wiki.mamedev.org/index.php?title=Submitting_Source_Code#Submitting_ROM_or_disk_dumps
You've already sent the files to me a while ago, but the melody ROM is different.

Follow this template when adding a new game to MAME (just edit your message above):
https://github.com/mamedev/build/blob/master/whatsnew/whatsnew_template.txt

So in this case:
New working systems
-------------------
Treasure Island (Tronica) [Milan Galcik, anyone else you want to credit]

MilanBA commented:
hap, thank you for your comment.
I've already submitted the ROM dump along with PCB photos to appropriate email address.

The melody ROM I sent you before contained a "bug" - there was a melody end marker missing at 0xAA. I've fixed it in the meantime and updated both hashes accordingly.

MilanBA commented:
Sure, done.

MilanBA commented:
Thanks hap.
One more thing - could you give a short explanation to the latest changes please? "d-pad is not 16way, put hour/min on service3/4"
So next time I could implement them automatically, knowing what to do and why.

MilanBA commented:
Well, you helped a lot with implementation of the "tripleh" some time ago, so the game is there thanks to your piece of code as well.
